src/components/Chat/BottomSheetGif.js

```diff
@@ -109,6 +121,7 @@ const BottomSheetGif = React.forwardRef((props, ref) => {
         refreshing={isLoading}
         onRefresh={onLoad}
         numColumns={2}
+        ListFooterComponent={<View style={{height: 100}} />}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a footer component to the list is a good way to provide some space at the end of the list, especially when the list is long. This can improve the user experience by making it easier to scroll to the last item in the list.
```diff
+ ListFooterComponent={<View style={{height: 100}} />}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->